src/cmd/lib/gen-secrets.ts

```diff
@@ -0,0 +1,77 @@
+import $RefParser from '@apidevtools/json-schema-ref-parser'
+import { writeFileSync } from 'fs'
+import { dump } from 'js-yaml'
+import { omit } from 'lodash-es'
+import { gucci, loadYaml } from '../../common/utils'
+
+const jsonPathToObj = (jsonPath: string, obj, value: string) => {
+  const arr = jsonPath.split('.')
+  let tmpObj = obj
```

Comment:
and why is this a copy by reference? This should not even be allowed by eslint @Dunky13 (can you make sure no shallow is turned on?).

Point is  it is a pointless assignment, and I suggest you learn javascript/typescript more, as we rely heavily on it...

Reply:
@Morriz if you are suggesting for me to learn javascript/typescript (which I know), then why did you hire me in the first place? I don't see any benefit in such comments...

Also, the reason for this assignment is actually ESLint complaining: https://eslint.org/docs/rules/no-param-reassign